### ISSUE TITLE
Inherited title not shown if description present

### ISSUE THREAD

**Issue by tcharding** [crywolf/mkchlog#18]
Multiple patches which all inherit title gives:
```
============================================

## Development

Internal development changes

* Add tfile

* Check for changes to the public API

* Make commits conform to mkchlog format

============================================
```
If we change the second patch to use `description:` the title gets dropped
```
============================================

## Development

Internal development changes

* Add tfile

Generate the full public API as a set of text files and commit them to the repository, for each subsequent PR do the same and check that the final state of the pubic API is not different. Implies that changing the public API requires a patch to the committed API text files.

* Make commits conform to mkchlog format

============================================
```
And if we add a title then we get a new subsection which breaks the list
```
============================================

## Development

Internal development changes

* Add tfile

### Check for changes to public API in CI

Generate the full public API as a set of text files and commit them to the repository, for each subsequent PR do the same and check that the final state of the pubic API is not different. Implies that changing the public API requires a patch to the committed API text files.

* Make commits conform to mkchlog format

============================================
```

### Expected behaviour

I was expecting to be able to inherit the title and get
```
============================================

## Development

Internal development changes

* Add tfile

* Check for changes to public API

   Generate the full public API as a set of text files and commit them to the repository, for each subsequent PR do the same and check that the final state of the pubic API is not different. Implies that changing the public API requires a patch to the committed API text files.

* Make commits conform to mkchlog format

============================================
```
or use a new title and get it as part of the list
```
============================================

## Development

Internal development changes

* Add tfile

* Check for changes to public API in CI

   Generate the full public API as a set of text files and commit them to the repository, for each subsequent PR do the same and check that the final state of the pubic API is not different. Implies that changing the public API requires a patch to the committed API text files.

* Make commits conform to mkchlog format

============================================
```

**Comment by tcharding** [comment]
FTR I did not render the markdown, I just guessed at what it should be.

**Comment by crywolf** [comment]
Thank you.

* Not shown title was fixed by 3b561f057421acf373e4dfb77c83a576bdbf0675
* And the "broken list" was meanwhile solved according to discussion in #10. Changes without description are displayed first.
